src/constants.js

```diff
@@ -1,4 +1,7 @@
 export const EMPTY_OBJ = {};
+export const EMPTY_VNODE = /** @type {import('./internal').VNode} */ (
+	EMPTY_OBJ
+);
```

Comment:
Probably unnecessary - will undo to reduce size later. Did this temporarily to make TypeScript happy in my editor.